README.md

```diff
@@ -354,25 +354,32 @@ easier.
 
   * `--no-pstate-check`: Do not crash out if Intel P-states are not disabled.
 
-  * `--hardware-reboots`: Restart physical hardware before each benchmark
-    execution.
-
 Note that you should not collect results intended for publication with
 development switches turned on.
 
+We also recommend that for 'real' benchmarking you turn on the `--hardware-reboots`
+switch to restart physical hardware before each benchmark execution.
+You will also need to ensure that Krun is restarted once the machine has
+rebooted. You can do this by hand, or by using the boot configuration provided
```

Comment:
What's a "boot configuration"?

Reply:
Good question, this was taking from an earlier commit, I guess it means rc.local?

Reply:
I sort-of guessed that... but it's a nonsense term, I think. Or, at least, I haven't heard it. So we probably need to expand it into more familiar English. I would personally suggest just saying "Depending on your OS, this might best be done by inserting a line into `/etc/rc.local`". I also wonder if we need to tell them *what* that line is, because I vaguely remember that you have to add some sort of "Krun is being started in resume mode" switch?

Reply:
It's more than one line I'm afraid. The rc.local snippet deals with adding the switch you mention (`--resume`).

Reply:
OK, so this may suggest that we need to do a bit more rejigging of the instructions?

Reply:
Here's the example rc.local btw: https://github.com/softdevteam/krun/blob/master/etc/rc.local.linux

Reply:
Yeah. Basically the instructions don't mention `rc.local` at all.

Reply:
OK. We can either fix that in this PR or open another PR. Might as well do it here...

Reply:
Maybe I'm being daft, but this conversation has confused me. We removed `--resume` and `--started-by-init` in a previous PR, when we added `--hardware-reboots` and simplified the CLI switches (at least, that's how I recall it, and I can't find those other switches in `--help`!). So, do you just want me to clarify the wording above, to remove "boot configuration", or is there much more to do here?

Reply:
Sarah is right. Those flags are now gone. Krun now uses the existence of a manifest file to decide if it is resuming or not.

I don't think you need to mention these implementation details.

Reply:
Right, so all we need to do, it sounds like, is make it clearer that you need to add *something* to `/etc/rc.local` or equivalent. Since this is OS / distribution dependent, maybe we could add a bullet list or subsection saying something like "on OpenBSD you might do this: XXX; on Debian this: XXX; on systemd: XXX YYY ZZZ AAA BBB CCC DDD" [sorry, I couldn't resist].